tbx/static_src/sass/config/_variables.scss

```diff
@@ -257,6 +259,7 @@ $grid-gutters-x-large: 120px;
 
 // Vertical spacing
 // Any changes here will also need to be made in tailwind.config.js
+$spacer-mini-minus: 10px;
```

Comment:
I suspect this one doesn't need to be a variable - it can just be a one-off spacing value.

Reply:
Resolved in new pr ( 6125861f0eb9389faa2bcf1f9a5b2f39dd25b1ae)